charts/argo-cd/templates/argocd-repo-server/deployment.yaml

```diff
@@ -87,6 +87,21 @@ spec:
           name: argocd-repo-server-tls
         - mountPath: /tmp
           name: tmp-dir
+        - mountPath: /helm-working-dir
+          name: helm-working-dir
+        - mountPath: /home/argocd/cmp-server/plugins
+          name: plugin
+        initContainers:
+        - command:
+          - cp
+          - -n
+          - /usr/local/bin/argocd
+          - /var/run/argocd/argocd-cmp-server
+          image: quay.io/argoproj/argocd:v2.2.0
```

Comment:
This probably needs to pick version from the variable

Reply:
huh. That wasn't intentional. Sorry bout that!
Must have been a copy-paste mistake that I overlooked before I submitted the PR

Reply:
No worries, thanks for the contribution :+1: